exchange/bidder.go

```diff
@@ -389,7 +389,15 @@ func (bidder *bidderAdapter) doTimeoutNotification(timeoutBidder adapters.Timeou
 			}
 		} else {
 			bidder.me.RecordTimeoutNotice(false)
+			if bidder.DebugConfig.TimeoutNotification.Log {
+				msg := fmt.Sprintf("TimeoutNotification: Failed to make timeout request: method(%s), uri(%s), error(%s)", toReq.Method, toReq.Uri, err.Error())
+				util.LogRandomSample(msg, glog.Warningf, bidder.DebugConfig.TimeoutNotification.SamplingRate)
+			}
 		}
+	} else if bidder.DebugConfig.TimeoutNotification.Log {
+		reqJSON, _ := json.Marshal(req)
```

Comment:
However unlikely it is for the marshaling to fail, please don't ignore the error response.